scripts/asm-installer/tests/run_basic_instance_template_creation

```diff
@@ -86,6 +86,7 @@ EOF
   verify_template_instance_with_custom_image
 
   cleanup_workload_service_account
+  kubectl delete workloadgroup "${WORKLOAD_NAME}" -n "${LT_NAMESPACE}"
```

Comment:
We must not make it required to do this for users before they delete their namespace. And I think we should test what our users can/should do. So this doesn't seem like a good idea to me. If we must do it to get tests passing we should open a P0 bug to revert it once we fix the underlying issue.

Reply:
It is only a flaky issue for the tests. I rerun the tests without these lines. It doesn't trigger the workloadgroup not deleted issues

Reply:
It is only a flaky issue for the tests. I rerun the tests without these lines. It doesn't trigger the workloadgroup not deleted issues

Reply:
We should definitely resolve the underlying issue. However submitting this to unblock tests sounds reasonable.